tools/ci/deps-ubuntu.sh

```
@@ -41,6 +41,16 @@ sudo apt-get -qq install -y \
   coinor-libcoinmp-dev \
   libhdf5-dev
 
+  sudo apt update
+  sudo apt-get install -y -V ca-certificates lsb-release wget
+  wget https://packages.apache.org/artifactory/arrow/$(lsb_release --id --short | tr 'A-Z' 'a-z')/apache-arrow-apt-source-latest-$(lsb_release --codename --short).deb
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)